Makefile

```diff
@@ -90,6 +90,11 @@ fmt-verify:
 	    exit 1; \
 	fi
 
+.PHONY: gomod-verify
+gomod-verify:
+	go mod vendor && go mod tidy
```

Comment:
I don't think `go mod vendor` is necessary for everyone.

Other than that, use `$(GO_CMD)` instead of directly `go`.

Reply:
yes, `go mod tidy` should suffice.